src/ralph/backends/storage/ldp.py

```diff
@@ -0,0 +1,142 @@
+"""OVH's LDP storage backend for Ralph"""
+
+import datetime
+import logging
+import sys
+
+import ovh
+import requests
+
+from ..mixins import HistoryMixin
+from .base import BaseStorage
+
+logger = logging.getLogger(__name__)
+
+
+class LDPStorage(HistoryMixin, BaseStorage):
+    """OVH's LDP storage backend"""
+
+    # pylint: disable=too-many-arguments
+
+    name = "ldp"
+
+    def __init__(
+        self,
+        endpoint=None,
```

Comment:
Why providing a default value ? The endpoint value looks like required and stored in a "private" property.

Reply:
This is tricky but storage-specific parameters should not be arguments but keywords.

Reply:
Yes but semantically this is wrong.

Reply:
In fact, making them arguments is not breaking the parameters discovery mechanism as you can call arguments as keywords in python.